openmetadata-ui/src/main/resources/ui/src/components/cardlist/CardListItem/CardWithListItem.tsx

```diff
@@ -34,6 +34,12 @@ const CardListItem: FunctionComponent<Props> = ({
   className,
   onRemove,
 }: Props) => {
+  const [isExpanded, setIsExpanded] = useState(isActive);
+
+  useEffect(() => {
+    setIsExpanded(isActive);
+  }, [isActive]);
```

Comment:
no, for expand and collapse we have to maintain state because my card can be active but collapse, in that case, isActive will be true but isExpanded will be false.